recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: 0a04f1d7d35a06cdff506593330d4414ea242c9172498ce191f5742eac499e17
 
 build:
-  skip: true  # [win or python_impl == 'pypy']
-  number: 1
+  skip: true  # [python_impl == 'pypy']
```

Comment:
If this does work, we could consider switching to `noarch: python` as well 🙂

Reply:
+1

Reply:
I think that's what we tried at the beginning on dask-mpi, and moved to this option because mpi4py wasn't reliable on Windows at the time.

Reply:
I can try both.  If both work, which is preferred?

Reply:
Yep we don't need to if it doesn't makes sense. Just an option 😉

Reply:
It looks like Microsoft-MPI hasn't been on conda-forge for very long (since Oct 2020), which is the reason we couldn't build dask-mpi initially on Windows (mpich and openmpi only build on linux and osx).